### PR TITLE
JW-529 cleanup

### PR DESCRIPTION
**Pull request by mclapinski**

### CODE REVIEW

**Comment by michal-clapinski** [comment]
Rebuild this please

**Comment by michal-clapinski** [comment]
Rebuild this please

**Comment by mclapinski** [comment]
Retest this please

File: src/ksync/ksync_tx_queue.cc [hunk 1]
```diff
@@ -63,13 +60,10 @@ void KSyncTxQueue::Init(bool use_work_queue) {
             (boost::bind(&KSyncSock::OnEmptyQueue, sock_, _1));
         return;
     }
-
-    #ifndef _WIN32
     assert((event_fd_ = eventfd(0, (FD_CLOEXEC | EFD_SEMAPHORE))) >= 0);
```
Comment (mclapinski) [inline src/ksync/ksync_tx_queue.cc]: I think that's a bug. Should be `EFD_CLOEXEC` instead of `FD_CLOEXEC`.

**Comment by mclapinski** [comment]
Retest this please

**Comment by mclapinski** [comment]
Retest this please

**Review by sodar — APPROVED** [review]

**Review by katrybacka — APPROVED** [review]

**Pull request merged**